test/extensions/filters/http/oauth2/filter_test.cc

```diff
@@ -946,6 +949,40 @@ TEST_F(OAuth2Test, OAuthTestFullFlowPostWithParameters) {
   filter_->finishFlow();
 }
 
+/**
+ * Testing oauth response after tokens are set.
+ *
+ * Expected behavior: cookies are set.
+ */
+TEST_F(OAuth2Test, OAuthAccessTokenSucessWithTokens) {
```

Comment:
Is this testing something that wasn't already being tested? Is this exercising the code added in this PR?

Reply:
One of the examples (wasm_cc) failed to verify, probably unrelated. Seems like you need to merge main to resolve the merge conflicts anyways, lets see if this still failing after that